nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/controllers/nf-ng-canvas-toolbox-controller.js

```diff
@@ -94,12 +94,21 @@ nf.ng.Canvas.ToolboxCtrl = function (processorComponent,
          * NOTE: The `component` must implement a dropHandler.
          */
         draggableComponentConfig: function(component) {
+
+            //add hover effect
+            component.getElement().hover(function () {
+                component.getElement().removeClass(component.icon).addClass(component.hoverIcon);
+            }, function () {
+                component.getElement().removeClass(component.hoverIcon).addClass(component.icon);
+            })
+
             return {
                 zIndex: 1011,
                 revert: true,
                 revertDuration: 0,
                 cancel: false,
                 containment: 'body',
+                cursor: 'grabbing',
```

Comment:
I can try this...but I don't think jQuery UI will like that....it is not like CSS in this case